tgui/packages/tgui/interfaces/SupermatterMonitor.js

```diff
@@ -102,33 +75,54 @@ const SupermatterMonitorDataView = (props, context) => {
           />
         }>
           <LabeledList>
-            <LabeledList.Item label="Core Integrity">
+            <LabeledList.Item label="Integrity">
               <ProgressBar
+                value={data.SM_integrity / 100}
                 ranges={{
-                  good: [95, Infinity],
-                  average: [80, 94],
-                  bad: [-Infinity, 79],
-                }}
-                minValue="0"
-                maxValue="100"
-                value={data.SM_integrity}>
-                {data.SM_integrity}%
-              </ProgressBar>
+                  good: [0.90, Infinity],
+                  average: [0.5, 0.90],
+                  bad: [-Infinity, 0.5],
+                }} />
             </LabeledList.Item>
             <LabeledList.Item label="Relative EER">
-              <Box color={powerToColor(data.SM_power)}>
-                {data.SM_power} MeV/cm3
-              </Box>
+              <ProgressBar
+                value={data.SM_power}
+                minValue={0}
+                maxValue={5000}
+                ranges={{
+                  good: [-Infinity, 5000],
+                  average: [5000, 7000],
+                  bad: [7000, Infinity],
+                }}>
+                {toFixed(data.SM_power) + ' MeV/cm3'}
+              </ProgressBar>
             </LabeledList.Item>
             <LabeledList.Item label="Temperature">
-              <Box color={temperatureToColor(data.SM_ambienttemp)}>
-                {data.SM_ambienttemp} K
-              </Box>
+              <ProgressBar
+                value={logScale(data.SM_ambienttemp)}
+                minValue={0}
+                maxValue={logScale(10000)}
+                ranges={{
+                  teal: [-Infinity, logScale(80)],
```

Comment:
@AffectedArc07 we support `teal:` right?

Reply:
Yes, see `CSS_COLORS` in `tgui/constants.js`

Reply:
Thanks!